.github/workflows/release.yaml

```diff
@@ -100,6 +111,36 @@ jobs:
           AC_CERTIFICATE_PASSWORD: ${{ secrets.AC_CERTIFICATE_PASSWORD }}
           AC_APIKEY_P8_BASE64: ${{ secrets.AC_APIKEY_P8_BASE64 }}
 
+      - name: Create release tag and release notes
+        run: |
```

Comment:
```suggestion
        run: |
          set -eu
```

Reply:
At least `set -e` is automatically enabled, but we can set it for explicitness.